calculator/parser.py

```diff
@@ -0,0 +1,137 @@
+BINARY_OPERATORS = ["+", "-", "*", "/"]
+
+
+def binary_operator(lhs, rhs, op):
+    """
+    Evaluates and acts on different binary operators
+
+    Parameters
+    ----------
+    lhs : integer or float
+        The first number to be operated on, will be on the left hand side of the op
+    
+    rhs : integer or float
+        The second number to be operated on, will be on the right hand side of the op
+    
+    op : string
+        The operator that is going to be applied to the numbers
+
+    Returns
+    -------
+    integer : The final answer after after the nums are operated on
+    """
+
+    if op == "+":
+        return lhs + rhs
+
+    elif op == "-":
+        return lhs - rhs
+
+    elif op == "*":
+        return lhs * rhs
+
+    elif op == "/":
+        return lhs / rhs
+    else:
+        raise ValueError("This operator is not supported, please try another one!")
+
+
+def push(e, stack):
+    """
+    Pushes the element (e) to the given stack, which mutates the stack
+
+    Parameters
+    ----------
+    e : element
+        The element to be pushed on top of the stack
+
+    stack : list
+        The stack that e is going to be pushed onto
+
+    Returns
+    -------
+    Null : returns nothing, only mutates the list   
+    """
+    return stack.append(e)
+
+
+def pop(stack):
+    """
+    Retrieves the last element from the given stack and deletes it, which mutates it
+
+    Parameters
+    ----------
+    stack : list
+        The stack (in list form) to be operated on
+
+    Returns
+    -------
+    Null : returns nothing only mutates the list
```

Comment:
Just don't return anything (e.g. remove the `return` portion of the statement). Also, if you're not returning anything, remove the Returns section of the docstring.